Lab_2_3_4_5/src/Grammar/Grammar.java

```diff
@@ -72,7 +72,7 @@ public void DisplayRequestedGrammarElement(int command) {
                 System.out.println(startingSymbol);
                 break;
             case 4:
-                DisplayStringList(productionList);
```

Comment:
Maybe it would have been more clear to also have had an UI? But ig this is also fine as the menu is related to the grammar only